client/src/components/Sub_RatingsReviews/FormComponents/ReviewModalContent.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import generateStars from '../generateStars.js';
+import RecommendedButton from './RecommendedButton.jsx';
+import RadioGroup from './RadioGroup.jsx';
+import AddReviewBody from './AddReviewBody.jsx';
+
+export default function ReviewModalContent({ setShowModal, product, reviewMetadata }) {
+  // STATES/CONSTANTS
+  const starInfo = {
+    1: 'Poor',
+    2: 'Fair',
+    3: 'Average',
+    4: 'Good',
+    5: 'Great',
+  };
+
+  const [reviewBody, setReviewBody] = useState({
+    product_id: product.id,
+    rating: 0,
+    summary: '',
+    recommend: false,
+    body: '',
+    name: '',
+    email: '',
+    photos: [],
+    characteristics: {},
+  });
+
+  const labels = {
+    Size: ['a size too small', 'half size too small', 'perfect', 'half size too big', 'a size too wide'],
+    Width: ['too narrow', 'slightly narrow', 'perfect', 'slightly wide', 'too wide'],
+    Comfort: ['uncomfortable', 'slightly uncomfortable', 'just ok', 'comfortable', 'perfect'],
+    Quality: ['very poor', 'below average', 'what i expected', 'pretty great', 'perfect'],
+    Length: ['runs short', 'runs slightly short', 'perfect', 'runs slightly long', 'runs long'],
+    Fit: ['runs tight', 'runs slightly tight', 'perfect', 'runs slightly long', 'runs long'],
+  };
+
```

Comment:
it seems you used labels again.  i wonder if you could store it in a separate js file and  just import it when you need it. just to avoid duplicate code.

Reply:
yeah i noticed this. will definitely refactor once functionality is complete.